ci-automation/vendor-testing/stackit.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) 2025 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+# Test execution script for STACKIT vendor.
+# This script is supposed to run in the mantle container.
+
+source ci-automation/vendor_test.sh
+
+copy_from_buildcache "images/${CIA_ARCH}/${CIA_VERNUM}/${STACKIT_IMAGE_NAME}" .
```

Comment:
We need to define the `STACKIT_IMAGE_NAME`, and `STACKIT_PARALLEL` in: ci-automation/ci-config.env